tests/e2e/elasticsearch/render.sh

```diff
@@ -8,16 +8,38 @@ if [ $IS_OPENSHIFT = true ]; then
 fi
 
 
+###############################################################################
+# TEST NAME: es-from-aio-to-production
+# DESCRIPTION: allInOne to production
```

Comment:
I think we can remove this

Reply:
@iblancasa with this header it is hard to identify the test

Reply:
I mean the `DESCRIPTION`

Reply:
ah ok, actually, I was confused about `aio`. then I looked into the steps and I got it `allInOne`. Hence I added this description :) 
if it looks odd, I can remove it.